### PR TITLE
add getActive orgs function

### PR DESCRIPTION
**Pull request by jay-meister**
getActive orgs needed for the add people drop down.

https://github.com/emfoundation/ce100-app/issues/442

The mock data has no useful active orgs without links, so the test is weak. I have manually tested the query but was hoping we could review our mock data as it is inefficient amending it in 4 different places.

### CODE REVIEW

**Comment by jay-meister** [comment]
sorry i need to update.

**Comment by jay-meister** [comment]
now updated and ready for review

File: example/server.js [hunk 1]
```diff
@@ -132,6 +132,19 @@ function init (config, callback) {
           },
           {
             method: 'GET',
+            path: '/orgsGetActive',
+            handler: function (request, reply) {
+              request.server.methods.pg.organisations.getActive(function (error, response) { // eslint-disable-line
```
Comment (SimonLab) [inline example/server.js]: I don't think you need eslint-disable-line here, I don't think the length of the line is too long

File: lib/index.js [hunk 2]
```diff
@@ -89,6 +89,16 @@ function register (server, options, next) {
     });
   };
 
+  var orgsGetActive = function (cb) {
+    return query(queries.orgsGetActive, pool, function (err, res) { //eslint-disable-line
```
Comment (SimonLab) [inline lib/index.js:108]: I don't think you need eslint-disable-line here, the line is not too long

**Review by SimonLab — APPROVED** [review]
LGTM, @mk4111 any comments?

**Review by mk4111 — COMMENTED** [review]
lgtm!

**Pull request merged**